code/__DEFINES/dcs/signals.dm

```diff
@@ -1176,6 +1176,7 @@
 #define COMSIG_TRY_STORAGE_RETURN_INVENTORY "storage_return_inventory"
 ///(obj/item/insertion_candidate, mob/user, silent) - returns bool
 #define COMSIG_TRY_STORAGE_CAN_INSERT "storage_can_equip"
+///(obj/item/contents_to_consume, int number_to_consume) -
```

Comment:
```suggestion
```
i assume you added this in? its loaded as an addition, we dont want to add to this file since we arent changing anything here